tutorials/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -486,22 +499,52 @@ Use the ``Godot.Collections.Dictionary<T>(Godot.Collections.Dictionary)`` constr
 Variant
 -------
 
-``System.Object`` (``object``) is used instead of ``Variant``.
+``Godot.Variant`` is used as the variant type, any variant-compatible type can be converted from/to it.
+Consider avoiding ``Godot.Variant`` and take advantage of C#'s type safety when possible,
+only use ``Godot.Variant`` when necessary to use Godot untyped APIs.
+
+The ``Variant.As{TYPE}`` methods or the generic ``Variant.As<T>`` method can be used to convert
+a ``Godot.Variant`` to a C# type. Since the ``Godot.Variant`` type contains implicit conversions
+defined for all the supported types calling these methods directly is usually not necessary.
```

Comment:
```suggestion
Any of ``Variant.As{TYPE}`` methods and a generic ``Variant.As<T>`` method can be used to convert
a ``Godot.Variant`` to a C# type. Since the ``Godot.Variant`` type contains implicit conversions
defined for all the supported types calling these methods directly is usually not necessary.
```

Reply:
This doesn't sound right to me, there is only one `Variant.As<T>` method so I feel like using `a` is wrong here.

Reply:
For some reason it trips me here, but I guess this is because I intuitively read "generic" in its literal sense, not as a CS term :) Sure, you can keep the article for the generic method (and the same for the snippet below). (but the rest of the suggestion stands).